new-packaging/src/common/neo4j.init

```diff
@@ -34,29 +38,51 @@
 
 PATH=/sbin:/usr/sbin:/bin:/usr/bin
 NAME=neo4j
-DAEMON=/usr/bin/$NAME
+DAEMON=/usr/bin/${NAME}
 PIDDIR=/var/run/${NAME}
 PIDFILE=${PIDDIR}/neo4j.pid
-SCRIPTNAME=/etc/init.d/$NAME-service
+SCRIPTNAME=/etc/init.d/${NAME}
```

Comment:
This looks surprising. Shouldn't this change have been made ages ago when we switched the service name from `neo4j-service`?